engine/ddf/src/waf_ddf.py

```diff
@@ -175,52 +176,73 @@ def compile_java_file(self, java_node, outdir):
     javac.set_inputs(java_node)
     javac.set_outputs(java_node_out)
 
+
+def get_protoc_cc_flags(self):
+    protoc_includes = [self.env['DYNAMO_HOME'] + '/share/proto', self.env['DYNAMO_HOME'] + '/ext/include']
+
+    if hasattr(self, "protoc_includes"):
+        protoc_includes = waflib.Utils.to_list(self.protoc_includes) + protoc_includes
+
+    protoc_flags = ""
+    for pi in protoc_includes:
+        if os.path.isabs(pi):
+            p = pi
+        else:
+            p = self.path.find_dir(pi).abspath()
+
+        protoc_flags += " -I %s " % p
+
+    return protoc_flags
```

Comment:
Slight difference between this fn and get_protoc_flags to get correct include paths for protoc